src/Runtime/OOPAggregateCallInterceptor.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * This file is part of the proophsoftware/event-machine.
+ * (c) 2017-2018 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventMachine\Runtime;
+
+use Prooph\EventMachine\Exception\RuntimeException;
+use Prooph\EventMachine\Messaging\Message;
+use Prooph\EventMachine\Messaging\MessageBag;
+use Prooph\EventMachine\Messaging\MessageFactory;
+use Prooph\EventMachine\Messaging\MessageFactoryAware;
+use Prooph\EventMachine\Runtime\OOP\AggregateAndEventBag;
+use Prooph\EventMachine\Runtime\OOP\Port;
+use Prooph\EventMachine\Util\DetermineVariableType;
+use Prooph\EventMachine\Util\MapIterator;
+
+final class OOPAggregateCallInterceptor implements CallInterceptor, MessageFactoryAware
```

Comment:
I agree because you used acronyms like `Json` and `Html` therefore I also think it should be `Oop` for consistency reasons. Just so that acronyms are handeled in a similar way.

Reply:
:+1: